pkg/apiserver/registry/core/shoot/storage/kubeconfig.go

```diff
@@ -85,6 +89,11 @@ func (r *KubeconfigREST) Create(ctx context.Context, name string, obj runtime.Ob
 		return nil, apierrors.NewBadRequest("no user in context")
 	}
 
+	groups, err := r.userGroupsFunc(ctx, userInfo, r.subjectAccessReviewer)
+	if err != nil {
+		return nil, err
```

Comment:
I was wondering if including some additional context to the returned `err` would be beneficial here. WDYT ?

```suggestion
		return nil, fmt.Errorf("failed to get user groups: %w", err)
```

Reply:
Yes, makes sense.